lib/collector/serverless.js

```diff
@@ -243,6 +247,29 @@ class ServerlessCollector {
     }
   }
 
+  /**
+   * Tests for writing to named pipe
+   */
+  shouldUsePipe() {
+    if (this.pipeError) {
+      return false
+    }
+    return fs.existsSync(this.pipePath)
+  }
+
+  /**
+   * Writes payload to pipe
+   */
+  flushToPipe(payload) {
```

Comment:
Just to be extra clear in javascript land... lets rename this to `flushToPipeSync` to indicate it is always going to be sync